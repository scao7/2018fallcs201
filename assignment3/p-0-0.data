4 8 358 ; 9 8 366 ; 7 0 206 ; 8 5 274 ; 9 4 254 ; 

