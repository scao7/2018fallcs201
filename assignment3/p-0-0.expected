0: 4
1: 8(4)358 9(4)254
2: 5(8)274
weight: 886
